### DISCUSSION TITLE
Tools for creating this image

### DISCUSSION THREAD

**Discussion by Neo42** [mobxjs/mobx#2845] [General] [answered]
Hi, @mweststrate. Sorry for bothering you but I'd like to create an image like this one but in Chinese for [this page](https://github.com/mobxjs/zh.mobx.js.org/blob/main/website/static/getting-started.html) on zh.mobx.js.org to bring better readability. What tools did you use to create this image? Thanks.

![image](https://user-images.githubusercontent.com/22409868/110741373-82a89880-826f-11eb-989d-d57e28187281.png)

**Comment by danielkcz** [comment]
I don't know which tools were used in this case, but I believe you can do this very easily with free [Canva](https://www.canva.com/).

**Comment by mweststrate** [comment, marked as answer]
I think InkScape
